src/method/method_connection_sr.cpp

```diff
@@ -80,13 +83,48 @@ namespace cubmethod
     free_and_init (buffer.ptr);
     return error;
   }
+#else
```

Comment:
looks better to add a comment for SA Mode

Reply:
I've added a comment